src/layout/Footer/Footer.jsx

```diff
@@ -0,0 +1,57 @@
+import "./Footer.css";
+
+const TEXT = {
+  codeit: "©codeit - 2023",
+  privacyPolicy: "Privacy Policy",
+  faq: "FAQ",
+  개인정보처리방침: "/privacy",
+  FAQ: "/faq",
```

Comment:
필요한 상수를 상수화해서 사용하신 부분 너무 좋습니다. 👍

다만, 몇가지 궁금한것도 개선점으로 보이는 부분이 있어 말씀드립니다.

궁금한점
- 개인정보처리방침만 한글 key로 관리하신 이유가 있을까요?

개선점
- 역할이 분리되면 좋을 듯합니다.
    - 현재 렌더링용 텍스트와 href 경로 역할이 혼재되어 있습니다.
    - 아래와 같이 용도에 맞게 응집화하면 관리하기 용이할듯합니다.
```suggestion
const TEXT= {
  codeit: "©codeit - 2023",
  privacyPolicy: "Privacy Policy",
  faq: "FAQ",
};
const LINK = {
  개인정보처리방침: "/privacy",
  FAQ: "/faq",
};
```